lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -311,21 +316,24 @@ function dosomething_kudos_disable_reactions($nid) {
 /**
  * Grab all Kudos data required to render the markup for a given reportback fid
  * @param  int $fid Reportback file id
+ * @param  int $total_rogue_kudos Total kudos from Rogue response (If Rogue is powering the gallery).
+ * @param  bool $liked_by_current_user Response from Rogue if the current user has reacted to the post or not.
  * @return array Values needed to render Kudos markup
  */
-function dosomething_kudos_get_kudos_data_for_fid($fid = NULL) {
+function dosomething_kudos_get_kudos_data_for_fid($fid = NULL, $total_rogue_kudos = NULL, $liked_by_current_user = NULL) {
   if (! $fid) {
     return [];
   }
-
   $kudos = [];
   $kudos['fid'] = $fid;
+  $kudos['existing_kids'] = dosomething_kudos_get_kudos_for_user_by_file($fid);
```

Comment:
That makes good sense to me! In the meantime, it might be nice to add a little `@TODO` comment.